cmd/lotus-storage-miner/retrieval-deals.go

```diff
@@ -160,80 +155,8 @@ var retrievalDealsListCmd = &cli.Command{
 	},
 }
 
-var retrievalSetAskCmd = &cli.Command{
```

Comment:
Why are we removing this command? Shouldn't this retrieval ask be the fallback price? It could be sent to the pricing strategy, or the pricing strategy could return a boolean saying whether it has calculated a price, or if it wants to fall back to the fixed price.